config/confd/templates/env.tmpl

```diff
@@ -75,6 +75,7 @@ TOKEN_AUTH_JWT_ALGO='{{getenv "TOKEN_AUTH_JWT_ALGO"}}'
 TRANSACTION_TIMEOUT_MS=30000
 UV_THREADPOOL_SIZE=128
 VPN_HOST={{getenv "VPN_HOST"}}
+{{if getenv "VPN_UDP_HOST"}}VPN_UDP_HOST="{{getenv "VPN_UDP_HOST"}"}"{{end}}
```

Comment:
I think there is a typo in `"}"}"` and I don't think we need the quotes around the `"{{getenv ... }}".
```suggestion
{{if getenv "VPN_UDP_HOST"}}VPN_UDP_HOST={{getenv "VPN_UDP_HOST"}}{{end}}
```

Moreover b/c of how `optionalVar` already handles empty strings as nulls, I think this could even (optionally) become:
```suggestion
VPN_UDP_HOST={{getenv "VPN_UDP_HOST"}}
```